ansible/playbooks/utils/check_volume.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Checking for necessary --extra_vars
+  hosts: all
+  vars:
+    example:
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
+
+  tasks:
+
+    - fail: msg="Must set VOLUME_DEVICE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE is undefined }}"
+
+    - fail: msg="Must set VOLUME_DEVICE_TYPE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE_TYPE is undefined }}"
+
+
+- name: Playbook to test volume before mounting
+  hosts: atmosphere
```

Comment:
All playbooks use `hosts: atmosphere` AFAIK. 

However it's clear there is ambiguity in the naming  -- perhaps another PR to rename `atmosphere` hosts to some variant of `atmosphere-vms` to avoid the ambiguity.